django_comparison_dashboard/graphs.py

```diff
@@ -191,6 +192,21 @@ def sankey(data, filter_set: SankeyGraphFilterSet):
 
         value.append(flow["value"])
 
+    node_colors = [
```

Comment:
Additionally, to setting colors via color dict, we could add color input forms (like in bar chart options) which can override/set color for each node.

Reply:
Do you mean a coloring for all nodes based on a specific column?
Should this then override the given color-dict as well?